tests/ProjectDeleteTest.php

```diff
@@ -3,24 +3,68 @@
 namespace Keboola\ManageApiTest;
 
 use Keboola\Csv\CsvFile;
-use Keboola\ManageApi\ClientException;
-use Keboola\StorageApi\Client;
+use Keboola\ManageApi\Backend;
+use Keboola\StorageApi\Components;
 use Keboola\StorageApi\Metadata;
+use Keboola\StorageApi\Options\Components as ComponentsOptions;
 use Keboola\StorageApi\Workspaces;
 
 class ProjectDeleteTest extends ClientTestCase
 {
-    public function testDeleteAndPurgeProjectWithData()
+    private const FILE_STORAGE_PROVIDER_S3 = 'aws';
+    private const FILE_STORAGE_PROVIDER_ABS = 'azure';
+
+    public function deleteAndPurgeProjectWithData(): \Generator
     {
+        return [
```

Comment:
Tady vracíš `array<\Generator>`. Je vidět, že tu nemáme phpstan. Zajímavé, že to funguje :D

Reply:
Uz jsem to pravil + tam pridal ten test na Snowfalke + ABS